CMakeLists.txt

```diff
@@ -640,6 +640,7 @@ if (BUILD_TESTING)
                         COMMAND
                         pytest
                         -x -n=${N} --reruns=2 --durations=10 --cache-clear -rpfsq
+			--junitxml=../../build/junit/${test_target}.xml
```

Comment:
Is it okay to assume that this is the build directory? Is there a way to get the actual build directory with cmake?

Reply:
This is the one being created by cmake both in [nix](https://github.com/aws/s2n-tls/blob/main/nix/shell.sh#L51) and [old school](https://github.com/aws/s2n-tls/blob/main/codebuild/bin/s2n_codebuild.sh#L76), so ./build is always a child of `s2n-tls`.